tests/e2e/e2e_test.go

```diff
@@ -12,18 +12,14 @@ import (
 	"testing"
 	"time"
 
-	"github.com/ava-labs/avalanche-network-runner/client"
```

Comment:
it's better to depend on github.com/ava-labs/avalanche-network-runner-sdk since github.com/ava-labs/avalanche-network-runner itself has avalanchego dependency (as does subnet-evm). github.com/ava-labs/avalanche-network-runner-sdk does not have any avalanchego dependency.